docs/README.md

```diff
@@ -180,6 +180,18 @@ module.exports = (webpackConfig, env) => {
 };
 ```
 
+### Output Filename
```

Comment:
Make `filename` lowercase please? The rest of the document doesn't capitalise words.

Reply:
https://github.com/pavelivanov/react-cosmos/commit/9da6a876f5d9d283d5103a50e4f714ac8484301c